web/workflow.R

```diff
@@ -145,8 +145,8 @@ if ('state.data.assimilation' %in% names(settings)) {
 # Run benchmarking
 if("benchmark" %in% names(settings)){
   status.start("BENCHMARKING")
-  settings <- papply(settings, function(x) create.benchmark(x, bety))
-  results <- papply(settings, function(x) calc.benchmark(x, bety))
+  settings <- papply(settings, function(x) create_benchmark(x, bety))
```

Comment:
1) There is no create_benchmark function, so this will throw an error
2) I don't think you want to be calling define_benchmark here because the benchmarks should already be defined before workflow.R is run